modules/lite/src/deckgl.js

```diff
@@ -80,7 +80,9 @@ export default class DeckGL extends Deck {
         width: '100%',
         height: '100%',
         canvas: deckCanvas,
-        controller: props.controller || Controller
+        controller: props.controller || Controller,
+        onViewStateChange: ({viewState}) =>
+          this._map && this._map.setProps({viewState}) && viewState
```

Comment:
Maybe comment what returning `viewState` does? It shouldn't be needed?

Reply:
`this._map.setProps` returns the map itself.